wpilibj/src/athena/java/edu/wpi/first/wpilibj/Preferences.java

```diff
@@ -84,12 +83,8 @@ private Preferences() {
    * Gets the vector of keys.
    * @return a vector of the keys
    */
-  public Vector getKeys() {
-    Vector<String> keys = new Vector<String>();
-    for (String key : m_table.getKeys()) {
-      keys.add(key);
-    }
-    return keys;
+  public Vector<String> getKeys() {
```

Comment:
I am going to leave this as is because it is not a breaking change but is there if teams want it.